graphql/types/paging.graphql

```diff
@@ -0,0 +1,8 @@
+type Paging {

```

Comment:
```suggestion
type Pagination {
```

Reply:
Actually, you don't need all this info, you just need the total, the rest can be either derived or it's the input that you are passing.